py/ns_verify_folder.py

```
@@ -52,45 +46,32 @@ def send_hook(message_content: str = '', PadPrint: bool = False):
             'content': message_content.strip()
         }
         headers = {"Content-type": "application/json"}
-        response = requests.post(WHOOK_URL, data=json.dumps(payload), headers=headers)
+        response = req_post(WHOOK_URL, data=json_dumps(payload), headers=headers)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)